scripts/test-many-configs.sh

```diff
@@ -0,0 +1,419 @@
+#!/bin/bash
+
+
+if [[ ${1} == "" ]]; then
+    echo "Please provide the name of your csv file with the format "URL,SHA,MODULE,tests1-9" on each line."
+    exit
+fi
+
+
+#0. File management/defining all helper functions
+
+flag=0  #Global flag designed to represent the status of the overall status of the whole build when running on a CI
+scriptDir=$(cd $(dirname $0); pwd)
+csvFile=$(cd $(dirname $1); pwd)/$(basename $1)
+cd ${scriptDir}
+
+if [[ ! -f ${csvFile} ]]; then
+    echo "No such csv exists in the given directory."
+    exit
+fi
+
+if [[ ! -d MC-script-results ]]; then
+    mkdir MC-script-results
+fi
+cd MC-script-results
+
+
+function setOriginalOrder() {
```

Comment:
Add some quick comment that this function copies the original order of tests we want to use in, as to avoid recomputing it each time (which can be different).